From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Niels Breet <niels.breet@jolla.com>
Date: Fri, 14 Jun 2019 12:42:06 +0300
Subject: [PATCH] Enable TrueType GX/AAT and OpenType table validation

---
 modules.cfg | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/modules.cfg b/modules.cfg
index fa8592b15171db74ed0f10e92fd42b3b146c2732..cc97ec1af50aebda824d5d7e06d65e35c167d074 100644
--- a/modules.cfg
+++ b/modules.cfg
@@ -117,7 +117,7 @@ AUX_MODULES += cache
 
 # TrueType GX/AAT table validation.  Needs `ftgxval.c' below.
 #
-# AUX_MODULES += gxvalid
+AUX_MODULES += gxvalid
 
 # Support for streams compressed with gzip (files with suffix .gz).
 #
@@ -136,7 +136,7 @@ AUX_MODULES += bzip2
 
 # OpenType table validation.  Needs `ftotval.c' below.
 #
-# AUX_MODULES += otvalid
+AUX_MODULES += otvalid
 
 # Auxiliary PostScript driver component to share common code.
 #
