From b8cafae81a4aa80b87f7d7cba808c4ba2955f5eb Mon Sep 17 00:00:00 2001
From: Niels Breet <niels.breet@jolla.com>
Date: Fri, 14 Jun 2019 12:42:06 +0300
Subject: [PATCH] Enable TrueType GX/AAT and OpenType table validation

---
 modules.cfg | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/modules.cfg b/modules.cfg
index d6cdbe523..181f91541 100644
--- a/modules.cfg
+++ b/modules.cfg
@@ -111,7 +111,7 @@ AUX_MODULES += cache
 
 # TrueType GX/AAT table validation.  Needs `ftgxval.c' below.
 #
-# AUX_MODULES += gxvalid
+AUX_MODULES += gxvalid
 
 # Support for streams compressed with gzip (files with suffix .gz).
 #
@@ -130,7 +130,7 @@ AUX_MODULES += bzip2
 
 # OpenType table validation.  Needs `ftotval.c' below.
 #
-# AUX_MODULES += otvalid
+AUX_MODULES += otvalid
 
 # Auxiliary PostScript driver component to share common code.
 #
-- 
2.29.2

